src/services/urlUtils.ts

```diff
@@ -122,41 +122,59 @@ namespace UrlUtils {
     });
   };
 
-  export const getNavigations = (request: UrlBeautifier.SearchUrlState) => {
+  // tslint:disable-next-line max-line-length
+  export const getNavigations = (state: Store.Indexed<Store.Navigation> | Store.AvailableNavigations, request: UrlBeautifier.SearchUrlState) => {
     return {
-      allIds: getAllIds(request),
-      byId: getById(request),
+      allIds: getAllIds(state, request),
+      byId: getById(state, request),
     };
   };
 
-  export const getAllIds = (request: UrlBeautifier.SearchUrlState) => {
-    return request.refinements.reduce((fields, refinement) => {
-      if (!fields.includes(refinement.field)) {
-        fields.push(refinement.field);
+  // tslint:disable-next-line max-line-length
+  export const getAllIds = (state: Store.Indexed<Store.Navigation> | Store.AvailableNavigations, request: UrlBeautifier.SearchUrlState) => {
```

Comment:
This change in signature is fine if we're sure nobody else is using it.

Reply:
we only import the `urlUtils` in `url` and don't export it

Reply:
(also we deleted these functions and I just brought them back)